lib/dor/workflow/client/lifecycle_routes.rb

```diff
@@ -78,3 +146,4 @@ def query_lifecycle(repo, druid, active_only:, version: nil)
     end
   end
 end
+# rubocop:enable Metrics/AbcSize, Metrics/MethodLength
```

Comment:
can you wrap this only arorund the methods that are too long? That is, I don't think `query_lifecycle` triggers this.

Reply:
Done.